### PR TITLE
bump operator to v1.3.6 for release-1.3

### PR DESCRIPTION
**Pull request by KanShiori**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: en/deploy-cluster-on-arm64.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Before starting the process, make sure that Kubernetes clusters are deployed on
 
 ## Deploy TiDB operator
 
-The process of deploying TiDB operator on ARM64 machines is the same as the process of [Deploy TiDB Operator in Kubernetes](deploy-tidb-operator.md). Note that with TiDB Operator versions earlier than v1.3.2, you should change the following configuration in the step [Customize TiDB operator deployment](deploy-tidb-operator.md#customize-tidb-operator-deployment): after getting the `values.yaml` file of the `tidb-operator` chart, you need to modify the `operatorImage` and `tidbBackupManagerImage` fields in that file to the ARM64 image versions. For example:
+The process of deploying TiDB operator on ARM64 machines is the same as the process of [Deploy TiDB Operator in Kubernetes](deploy-tidb-operator.md). Note that with TiDB Operator versions earlier than v1.3.1, you should change the following configuration in the step [Customize TiDB operator deployment](deploy-tidb-operator.md#customize-tidb-operator-deployment): after getting the `values.yaml` file of the `tidb-operator` chart, you need to modify the `operatorImage` and `tidbBackupManagerImage` fields in that file to the ARM64 image versions. For example:
```
Comment (ran-huang) [inline en/deploy-cluster-on-arm64.md:16]: 这里以前写错了吗？

Comment (KanShiori) [reply en/deploy-cluster-on-arm64.md:16]: 对，中文版是 1.3.1，估计发版时候弄错了

Comment (KanShiori) [reply en/deploy-cluster-on-arm64.md:16]: 看了下应该是 1.3.1

**Review by ran-huang — APPROVED** [review]

**Pull request merged**